requirements/project-requirements.txt

```diff
@@ -86,6 +86,7 @@ knock-subdomains==7.0.1
 dotnetfile==0.2.4
 docxpy==0.8.5
 pylnk3==0.4.2
+androguard==3.4.0a1
```

Comment:
Using version >=4.x will result in dependency conflict as follows:
` INFO: pip is looking at multiple versions of quark-engine to determine which version is compatible with other requirements. This could take a while.`
`30.49 ERROR: Cannot install -r project-requirements.txt (line 70) and androguard==4.1.2 because these package versions have conflicting dependencies.`
`30.49 The conflict is caused by:`
`30.49     The user requested androguard==4.1.2 and quark-engine 24.10.1 depends on androguard==3.4.0a1`
`30.49 To fix this you could try to:`
 `1. loosen the range of package versions you've specified`
 `2. remove package versions to allow pip to attempt to solve the dependency conflict`

All the functions that you've mentioned can be implemented in the similar way in version `3.4.0a1` as well

Reply:
thanks for the explanation.

Could you please add this information as a comment here? That would help the maintainers for the future